yt_dlp/extractor/tiktok.py

```diff
@@ -46,7 +46,8 @@ def _get_sigi_state(self, webpage, display_id):
     def _real_initialize(self):
         if self._session_initialized:
             return
-        self._request_webpage(HEADRequest('https://www.tiktok.com'), None, note='Setting up session', fatal=False)
+        self._request_webpage(HEADRequest('https://www.tiktok.com'), None,
+                              headers={'User-Agent': 'User-Agent:Mozilla/5.0'}, note='Setting up session', fatal=False)
```

Comment:
```suggestion
                              headers={'User-Agent': 'Mozilla/5.0'}, note='Setting up session', fatal=False)
```
typo?

Reply:
this is deliberate; seemingly any HTTP/1.1 request to `tiktok.com` will now timeout unless the UA is exactly `User-Agent:Mozilla/5.0` (see also line 556)

although someziggyman looks to be having an issue even with this UA, hmm